modules/apps/account/account-admin-web/src/main/resources/META-INF/resources/account_users_admin/js/AccountUsersAdminManagementToolbarPropsTransformer.js

```diff
@@ -106,21 +86,73 @@ class ManagementToolbarDefaultEventHandler extends DefaultEventHandler {
 			title: dialogTitle,
 			url: accountEntrySelectorURL,
 		});
-	}
+	};
 
-	_updateAccountUsers(url) {
-		const form = this.one('#fm');
+	const selectAccountEntries = (itemData) => {
+		_openAccountEntrySelector(
+			Liferay.Language.get('select'),
+			`${portletNamespace}selectAccountEntries`,
+			Liferay.Language.get(itemData.dialogTitle),
+			itemData.accountEntriesSelectorURL,
+			(selectedItems) => {
+				var redirectURL = Liferay.Util.PortletURL.createPortletURL(
```

Comment:
We can use `createPortletURL`, like in `addAccountUser`. It is already imported.